README.md

```diff
@@ -215,14 +158,9 @@ of the features in the library.
 
 ### Document Parsing
 
-The easiest way to parse a document in unstructured is to use the `partition` brick. If you
-use `partition` brick, `unstructured` will detect the file type and route it to the appropriate
-file-specific partitioning brick.
```

Comment:
>If you
use `partition` brick, `unstructured` will detect the file type and route it to the appropriate
file-specific partitioning brick.

This is moved below the code snippet, where it immediately prefaces and contextualizes the table that displays supported file types and their corresponding partition function.